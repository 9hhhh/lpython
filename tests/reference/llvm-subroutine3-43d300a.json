{
    "basename": "llvm-subroutine3-43d300a",
    "cmd": "lfortran --show-llvm {infile} -o {outfile}",
    "infile": "tests/subroutine3.f90",
    "infile_hash": "a524604ef3f17ed8a7b81b5ac3b03932cbc3ec6f308982532b4d9900",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-subroutine3-43d300a.stdout",
    "stdout_hash": "b891b72e40a4eea8c83c849e935954d61188ff8db056044dac513d91",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}