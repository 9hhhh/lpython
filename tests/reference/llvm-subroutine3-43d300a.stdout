; ModuleID = 'LFortran'
source_filename = "LFortran"

define i64 @f() {
.entry:
  %f = alloca i64
  store i64 42, i64* %f
  %0 = load i64, i64* %f
  ret i64 %0
}

